app/component/itinerary/navigator/NaviCardContainer.js

```diff
@@ -1,47 +1,55 @@
-import React, { useEffect, useState, useRef } from 'react';
-import PropTypes from 'prop-types';
-import { intlShape } from 'react-intl';
 import distance from '@digitransit-search-util/digitransit-search-util-distance';
 import { matchShape, routerShape } from 'found';
-import { legShape, configShape } from '../../../util/shapes';
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import { intlShape } from 'react-intl';
 import { legTime, legTimeStr } from '../../../util/legUtils';
+import { configShape, legShape } from '../../../util/shapes';
```

Comment:
Just curious what's happening here, just reordering imports?

Reply:
Yup, just reordering thanks to VS Code and `Ctrl+O`